source/installation.rst

```diff
@@ -18,13 +18,26 @@ means that it will need:
 - your LDAP URL, base DN, and attribute to search for (in some rare cases a
   bind DN and corresponding bind password)
 
+Building From Source
+--------------------
```

Comment:
Can this section be under the TOC tree? Let's promote the more common use case.

Reply:
Made a change to move this to the last item in the TOC tree for installs. Let me know if that looks like what you were thinking.